sys/include/net/sock/ip.h

```diff
@@ -316,9 +316,10 @@ typedef struct sock_ip sock_ip_t;
  *          elsewhere
  * @return  -EAFNOSUPPORT, if `local != NULL` or `remote != NULL` and
  *          sock_ip_ep_t::family of @p local or @p remote is not supported.
- * @return  -EINVAL, if `proto` is not supported or if sock_ip_ep_t::netif of
```

Comment:
`proto` not supported is already covered by return value `-EPROTONOSUPPORT` documented below, so I removed that aspect.